main.py

```diff
@@ -1,7 +1,58 @@
 import init_django_orm  # noqa: F401
 
+from db.models import Genre, Actor
 from django.db.models import QuerySet
 
 
 def main() -> QuerySet:
-    pass
+    genres = (
+        "Western",
+        "Action",
+        "Dramma"
+    )
+
+    actors = (
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    )
+
+    [Genre.objects.create(name=genre) for genre in genres]
+    [Actor.objects.create(
+        first_name=name,
+        last_name=surname
+    ) for name, surname in actors]
```

Comment:
List comprehensions in Python are generally used for creating new lists. Using regular loops will be more clear here